nix/sources.json

```diff
@@ -70,15 +70,15 @@
         "version": "v1.2.0"
     },
     "motoko-base": {
-        "branch": "next-moc",
```

Comment:
This is to be changed back to `next-moc` once https://github.com/dfinity/motoko-base/pull/589 is merged.